modules/ROOT/pages/websocket/websocket-connector.adoc

```diff
@@ -0,0 +1,97 @@
+= WebSocket Connector - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Support Category: Select
+
+WebSocket Connector Version 1.0
+
+A WebSocket is a bi-directional, TCP connection between a client and server that is kept open until the application closes it. When using WeSockets, an application makes an initial connection with HTTP. The connection then gets upgraded to a TCP socket-based connection.
+
+WebSockets are a key piece of modern integration due to their ability to:
+
+* Establish bi-directional and full-duplex communication between a server and client
+* Implement server push notifications
+* Seamlessly work on top of existing HTTP infrastructure
+* Avoid the need for clients to regularly poll the server for new data
+
+WebSocket Connector enables you to create real-time applications that need to avoid the overhead causes by the HTTP request-response message exchange pattern.  <<TBD - I will probably add more text here once I get this information from Esteban.>>
+
+== Common Use Cases
+
+Typical use cases for WebSocket Connector include chat applications and stock quote applications.
+
+=== Chat Applications
+
+A chat application sends messages to one or more groups to which multiple clients subscribe. If a Mule app acting as a server uses HTTP to send messages, the clients must constantly poll the server for available messages, which can create performance issues. If an app uses WebSockets instead, the server can push the available message to all subscribed users without the need for client polling.
+
+=== Stock Quotes Applications
```

Comment:
This is not helpful

Reply:
This was in the original as far as I can tell.